python/mleap/pyspark/feature/math_binary.py

```diff
@@ -57,15 +60,18 @@ def __init__(
         :param inputA: column name for the left side of operation (string)
         :param inputB: column name for the right side of operation (string)
         :param outputCol: output column name (string)
+        :param defaultA: Default to use instead of inputA
```

Comment:
So `inputA` represents the column name but `defaultA` is the value for that column? Can we make this more clear?

Reply:
Yes, I can make this more clear. Basically, when you provide defaultA, you should skip inputA. This is useful for cases where you want to e.g., add +1 to all entries in a col or mutliply a col by 3. 
I had earlier thought about making assertions to make sure that users provide either inputA or defaultA but not both, but thought against doing so to keep parity with the scala transformer